scripts/scil_assign_uniform_color_to_tractogram.py

```diff
@@ -38,25 +35,14 @@ def _build_arg_parser():
     p.add_argument('in_tractograms', nargs='+',
```

Comment:
Should the script name be `..._to_tractograms.py` since it can take more than 1 input?